compat/maven-embedder/src/test/java/org/apache/maven/cli/MavenCliTest.java

```diff
@@ -105,6 +102,14 @@ void tearDown() throws Exception {
         }
     }
 
+    // Helper method for containsExactlyInAnyOrder assertion
```

Comment:
Why reinvent the wheel when this has already been done in other frameworks?
better to focus on release version, those changes have 2 effects:
- delay release as you have to waste time on that
- doesn't bring any added value to user and make developer life more complicated.